Logger/BaseLogger.cs

```diff
@@ -2,6 +2,9 @@
 {
     public abstract class BaseLogger
     {
+        public string? Name { get; set; }
```

Comment:
```suggestion
        public string? ClassName { get; set; }
```
A more obvious name. And my `ctrl+f` will pick it up. :)